### PR TITLE
Feature/practice two

### PR DESCRIPTION
**Pull request by nquan1007**

### CODE REVIEW

File: Practice-Two/src/partials/questions.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                        <figure>
+                            <img src="./assets/images/questions/questions-down-arrow.png" alt="questions-down-arrow" />
+                        </figure>
+                    </div>
+                </article>
+                <article class="questions-item">
+                    <div class="questions-item-title">
+                        <p class="questions-item-description-desktop">How to return products?</p>
+                        <picture>
+                            <source media="(min-width:768px)" srcset="./assets/images/questions/questions-up-arrow.png" />
```
Comment (MinhNguyen41092) [inline Practice-Two/src/partials/questions.html]: Need to explain this

Comment (nquan1007) [reply Practice-Two/src/partials/questions.html]: Did you mean I have to explain the way I do or is there any problem with this sectioon?

File: Practice-Two/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Nguyen Dinh Minh
```
Comment (MinhNguyen41092) [inline Practice-Two/LICENSE]: It's your name here.

File: Practice-Two/src/styles/base/reset.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
```
Comment (MinhNguyen41092) [inline Practice-Two/src/styles/base/reset.css]: Indentation should be 2 spaces.

File: Practice-Two/src/styles/components/button-styles.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
+.btn-primary {
```
Comment (MinhNguyen41092) [inline Practice-Two/src/styles/components/button-styles.css]: Unnecessary `-styles` in file name.

File: Practice-Two/src/styles/components/index.css [hunk 5]
```diff
@@ -0,0 +1,4 @@
+@import url(button-styles.css);
+@import url(content-styles.css);
+
```
Comment (MinhNguyen41092) [inline Practice-Two/src/styles/components/index.css]: Redundant empty line.

File: Practice-Two/src/styles/layouts/contact.css [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+  border: none;
+  border-bottom: 1px solid var(--color-primary);
+  font-weight: var(--font-weight-title);
+  font-size: var(--font-size-secondary);
+  line-height: var(--line-height-subtitle);
+  margin-bottom: 29px;
+}
+
+.contact-form-query {
+  border-bottom: 1px solid #bdbdbd;
```
Comment (MinhNguyen41092) [inline Practice-Two/src/styles/layouts/contact.css]: Create variable for color.

File: Practice-Two/src/partials/header.html [hunk 7]
```diff
@@ -0,0 +1,35 @@
+<header class="header-nav">
+  <h1>
+    <a href="#">
+      <img
+        src="./assets/images/header/header-logo.png"
+        alt="header-logo"
```
Comment (MinhNguyen41092) [inline Practice-Two/src/partials/header.html]: alt should be your website name, otherwise it's meaningless.

**Review by MinhNguyen41092 — APPROVED** [review]

**Pull request merged**